benchmarks/benches/transaction.rs

```diff
@@ -1,69 +1,159 @@
-use benchmarks::slow_config;
-use criterion::{criterion_group, criterion_main, Criterion};
+use benchmarks::{slow_config, very_slow_config};
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
 use ironfish_rust::{
     assets::{asset::Asset, asset_identifier::NATIVE_ASSET},
     test_util::make_fake_witness,
-    Note, ProposedTransaction, SaplingKey,
+    transaction::batch_verify_transactions,
+    Note, ProposedTransaction, SaplingKey, Transaction,
 };
 
 pub fn simple(c: &mut Criterion) {
-    let key = SaplingKey::generate_key();
-    let public_address = key.public_address();
+    c.bench_function("transaction::simple", |b| {
```

Comment:
The 2 existing tests are unchanged, besides moving their setup code into the setup block using `iter_batched`